lib/data/conversationMessage.ts

```
@@ -423,7 +425,7 @@ export const ensureCleanedUpText = async (
 ) => {
   if (message.cleanedUpText !== null) return message.cleanedUpText;
   const cleanedUpText = generateCleanedUpText(message.body ?? "");
-  await tx.update(conversationMessages).set({ cleanedUpText }).where(eq(conversationMessages.id, message.id));
+  await tx.update(conversationMessages).set({ cleanedUpText, encryptedCleanedUpText: cleanedUpText }).where(eq(conversationMessages.id, message.id));
```

ESLint

[error] Replace `.update(conversationMessages).set({·cleanedUpText,·encryptedCleanedUpText:·cleanedUpText·})` with `⏎····.update(conversationMessages)⏎····.set({·cleanedUpText,·encryptedCleanedUpText:·cleanedUpText·})⏎····`

(prettier/prettier)